easybuild/easyconfigs/p/PRC/PRC-1.5.6-intel-2018a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'PRC'
+version = '1.5.6'
+
+homepage = 'http//supfam.org/PRC/'
+description = """ PRC is a stand-alone program for aligning and scoring two profile hidden Markov models.
+ This can be used to detect remote relationships between profiles more effectively than by doing simple
+ profile-sequence comparisons. PRC takes into account all transition and emission probabilities in both
+ hidden Markov models. """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://supfam.org/PRC/programs/PRC-%(version)s/']
+sources = ['%(namelower)s-%(version)s-src.tgz']
+checksums = ['2ad9dcd0621a7262b5754426a62cd630dfbf14b04481fde116fa169d89249394']
+
+postinstallcmds = ["mkdir -p %(installdir)s/bin && cp prc convert_to_prc merge_aligns.pl %(installdir)s/bin"]
```

Comment:
@smoors So `make install` doesn't do anything?

Reply:
no, it's not even defined in the makefile

Reply:
@smoors Then we should skip the `install` step (and maybe include a comment to explain why we're using this approach):

```python
skipsteps = ['install']
```